kong.conf.default

```diff
@@ -1357,57 +1357,52 @@
-                                 #              runtime for running malicious code, therefore,
-                                 #              you should properly protect your Admin API endpoint
-                                 #              even with sandboxing enabled. The sandbox only
-                                 #              provides protection against trivial attackers or
-                                 #              unintentional modification of the Kong global
-                                 #              environment.
-                                 # - `on`: allow loading of Lua functions from admin
-                                 #         supplied sources and do not use a sandbox when
-                                 #         executing them. Functions will have unrestricted
+                                 #              See also `untrusted_lua_sandbox_requires` and
+                                 #              `untrusted_lua_sandbox_environment` below for
+                                 #              furrther customisation.
+                                 #
+                                 # - `on`: functions will have unrestricted
                                  #         access to global environment and able to load any
                                  #         Lua modules. This is similar to the behavior in Kong
                                  #         prior to 2.3.0.
-                                 #
-                                 #         LuaJIT bytecode loading is disabled.
-
-#untrusted_lua_sandbox_requires =
-                                 # Comma-separated list of modules allowed to be loaded
-                                 # with `require` inside the sandboxed environment. Ignored
-                                 # if `untrusted_lua` is not `sandbox`.
-                                 #
-                                 # Note: certain modules, when allowed, may cause sandbox
-                                 # escaping trivial.
 
-#untrusted_lua_sandbox_environment =
-                                 # Comma-separated list of global Lua variables
-                                 # that should be made available inside the sandboxed
-                                 # environment. Ignored if `untrusted_lua` is not `sandbox`.
-                                 #
-                                 # Note: certain variables, when made available,
-                                 # may cause sandbox escaping trivial.
+#untrusted_lua_sandbox_requires = # Comma-separated list of modules allowed to be loaded
+                                  # with `require` inside the sandboxed environment. Ignored
+                                  # if `untrusted_lua` is not `sandbox`.
+                                  #
+                                  # Note: certain modules, when allowed, may cause sandbox
+                                  # escaping trivial.
```

Comment:
not sure if this sentence is parseable.  what about:
```suggestion
                                  # Note: certain modules, when allowed, may allow trivial
                                  # sandbox escaping.
```

maybe adding example "dangerous" module, like `os`, `luaposix`?

Reply:
Noted, attempted to rephrase - hope I understood what it was trying to say correctly.